button.html

```diff
@@ -0,0 +1,16 @@
+<!Doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title> button magic </title>
+    <script> src = "./buttoncount.js"</script>
+</head>
+
+<body>
+    
+    <button onclick="fun()" > click me </button>
+    <div> clicks: <a id ="count"> 0 </a></div>
```

Comment:
Why have you used <a> tag here? What is the reason behind it?

Reply:
i removed the anchor tag .